archive/j/javascript/longest-word.js

```diff
@@ -0,0 +1,22 @@
+const error_msg = "Usage: please provide a string";
+const inputString = process.argv[2].trim().replace(/\\n|\\t|\\r/g, ' ')
+const words = inputString.split(' ')
```

Comment:
These should be done after checking if there are enough command-line arguments.

Reply:
Also, I'd recommend putting the `trim` after the `replace` to remove leading and trailing spaces.

Reply:
Thanks @rzuckerm for the suggestions, I have made the changes to the code.